packages/scripts/src/node-environment.ts

```diff
@@ -71,6 +82,7 @@ export async function runNodeEnvironment({
             }
         }
     }
+    clear();
```

Comment:
why do you clear here?

Reply:
I thought that after finishing to evaluate the external features, we no longer need to preserve this mapping